packages/core-blocks/src/style.scss

```diff
@@ -0,0 +1,125 @@
+@import './audio/style.scss';
+@import './block/edit-panel/style.scss';
+@import './block/indicator/style.scss';
+@import './button/style.scss';
+@import './categories/style.scss';
+@import './columns/style.scss';
+@import './cover-image/style.scss';
+@import './embed/style.scss';
+@import './file/style.scss';
+@import './gallery/style.scss';
+@import './image/style.scss';
+@import './latest-comments/style.scss';
+@import './latest-posts/style.scss';
+@import './paragraph/style.scss';
+@import './pullquote/style.scss';
+@import './quote/style.scss';
+@import './separator/style.scss';
+@import './subhead/style.scss';
+@import './table/style.scss';
+@import './text-columns/style.scss';
+@import './video/style.scss';
+
+.has-pale-pink-background-color {
```

Comment:
Should we put colors in its own file? I'm also not quite sure why they are located in this package?